content/en/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,151 @@
+---
+reviewers:
+- zparnold
+title: Overview of Cloud Native Security
+content_template: templates/concept
+weight: 1
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+Kubernetes Security (and security in general) is an immense topic that has many
+highly interrelated parts. In today's era where open source software is
+integrated into many of the systems that help web applications run,
+we need some overarching concepts to help guide our intuition about how we can
+think about security holistically. This guide will define a thought-framework for
+what we think of when we imagine "Kubernetes Security." The thought framework we selected is completely arbitrary
```

Comment:
Are you writing a guide?
thought-framework vs thought framework?

Reply:
Yes it's both. A guide for Kubernetes in specific, a thought-framework for software security in general (of which Kubernetes is a subset)

Reply:
- either use thought-framework or just `thought framework`
- this sounds a bit `jargony`

Reply:
:joy: That's the only way I know how!